libs/api/domains/license-service/src/lib/graphql/genericLicense.model.ts

```diff
@@ -41,6 +41,9 @@ export class GenericLicenseProvider {
     description: 'ID of license provider',
   })
   id!: GenericLicenseProviderId
+
+  @Field({ description: 'Display name of license' })
+  name?: string
```

Comment:
Shouldn't this be defined for `GenericLicense` not `GenericLicenseProvider`?
On `GenericLicense` we could also add the slug

Reply:
Moved